src/intl/resolver.js

```diff
@@ -329,9 +325,9 @@ function Pattern(env, ptn) {
             `(${str.length}, max allowed is ${MAX_PLACEABLE_LENGTH})`
           )
         );
-        result += FSI + str.substr(0, MAX_PLACEABLE_LENGTH) + PDI;
+        result += str.substr(0, MAX_PLACEABLE_LENGTH);
```

Comment:
There's no difference between `+=` and `.join()`.  Also no difference between using `FSI` and `PDI` here and not.  I tested using `workload-high.ftl`.